### PR TITLE
add tests for #4363

### PR DESCRIPTION
**Pull request by andriidovhan**

### CODE REVIEW

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy [hunk 1]
```diff
@@ -354,6 +359,100 @@ class SwitchSyncSpec extends BaseSpecification {
...
+    @Tags([VIRTUAL, SMOKE])
+    def "Able to synchronize misconfigured flow meter"() {
+        given: "An active switch with flow on it"
+        def sw = topology.activeSwitches.first()
+        def flow = flowHelperV2.singleSwitchFlow(sw)
+        flowHelperV2.addFlow(flow)
+
+        when: "Update flow's meter"
+        def flowMeterIdToManipulate = northbound.getAllMeters(sw.dpId).meterEntries.find {
+            it.meterId > MAX_SYSTEM_RULE_METER_ID
```
Comment (niksv) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy:407]: I think 1meterId >= MIN_FLOW_METER_ID` will be more correct

**Review by niksv — APPROVED** [review]

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy (hunk 1 above)
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy]: fixed

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy [hunk 2]
```diff
@@ -354,6 +359,100 @@ class SwitchSyncSpec extends BaseSpecification {
...
+            with(northbound.validateSwitch(sw.dpId)) {
+                it.meters.misconfigured.empty
+                it.meters.proper.find {
+                    it.meterId == flowMeterIdToManipulate.meterId
+                }.burstSize == flowMeterIdToManipulate.burstSize
+                it.meters.proper.find {
+                    it.meterId == flowMeterIdToManipulate.meterId
+                }.rate == flowMeterIdToManipulate.rate
+            }
+        }
```
Comment (rtretyak) [inline src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy:447]: Is it fixed my re-installing meters + rules or by 'update' command?

**Review by rtretyak — APPROVED** [review]

File: src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy (hunk 2 above)
Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy:447]: by 'update' command I guess.
I will sync with dev on this question

Comment (andriidovhan) [reply src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchSyncSpec.groovy:447]: I've synced with dev, it is fixed by 'updated'

**Pull request merged**